README.md

```diff
@@ -17,9 +17,14 @@ handler) error when linking (tested in Visual Studio 2017). (TODO: build script?
 
 ## Building
 We use CMake as the cross-platform build system.
+
 ```bash
-git checkout https://github.com/Mindwerks/XLEngine.git;
-mkdir build; cd build; cmake ../XLEngine; make -j4
```

Comment:
These instructions are actually correct. Note that it doesn't `cd XLEngine` after cloning, so `build` is a sibling directory and `cmake ../XLEngine` is correct to make an out-of-tree build. The updated instructions are correct too though, so I'll leave the decision of which to have up to @psi29a.

Reply:
Yeah, when developing... it's always been a best practice of mine to have your out-of-tree build. Yes there are tricks you can do with .gitignore, but it is still messy.